doc/docs/Exploiting_Symmetry.md

```diff
@@ -74,3 +74,87 @@ One subtlety that arises in specifying the symmetry of the system is that you ha
+
+The simulation script is in [examples/grid_discretization.py](https://github.com/NanoComp/meep/blob/master/python/examples/grid_discretization.py).
+
+```py
+import numpy as np
+import meep as mp
+
+resolution = 50 # pixels/um
+
+sx = 1.5
+sy = 6.0
+cell_size = mp.Vector3(sx,sy)
+
+dpml = 1.0
+pml_layers = [mp.PML(direction=mp.Y,
+                     thickness=dpml)]
+
+fcen = 1.0
+
+def src_amp_func(m):
+    def _src_amp_func(p):
+        return np.cos(2*np.pi*m*(p.x+0.5*sx)/sx)
+    return _src_amp_func
+
+sources = [mp.Source(mp.GaussianSource(fcen,fwidth=0.2*fcen),
+                     component=mp.Ez,
+                     center=mp.Vector3(0,-0.5*sy+dpml),
+                     size=mp.Vector3(sx,0),
+                     amp_func=src_amp_func(6))]
+
+sim = mp.Simulation(cell_size=cell_size,
+                    resolution=resolution,
+                    k_point=mp.Vector3(),
+                    boundary_layers=pml_layers,
+                    sources=sources,
+                    default_material=mp.Medium(index=3.5))
+
+flux_mon = sim.add_flux(fcen, 0, 1,
+                        mp.FluxRegion(center=mp.Vector3(0,0.5*sy-dpml),size=mp.Vector3(sx)))
+
+sim.run(until_after_sources=20)
+
+flux = mp.get_fluxes(flux_mon)
+print("flux:, {}".format(flux[0]))
+```
+
+Plotting the radiated flux as a function of the resolution (using a semi-logarithmic scale) reveals an unexpected finding.
+
+<center>
+![](images/grid_break_mirror_symmetry.png)
+</center>
+
+The radiated flux of the evanescent mode varies by nearly twelve orders of magnitude when the resolution changes by just one! Also, the radiated flux is "zero" (the correct/expected result) only when the resolution is an even number. These large variations are due to the cell dimensions in the periodic $x$ direction which are displayed in the simulation output:
+
+```
+Computational cell is 1.5 x 6 x 0 with resolution 50
+flux:, 6.788984186414434e-16
+...
+Computational cell is 1.5098 x 6 x 0 with resolution 51
```

Comment:
i.e. it is changing the period, as I said.

It seems like this is something that should be fixed.